pkgs/development/python-modules/vcard/default.nix

```
@@ -11,20 +11,18 @@
 
 buildPythonPackage rec {
   pname = "vcard";
-  version = "0.16.1";
+  version = "1.0.0";
   pyproject = true;
 
   disabled = pythonOlder "3.8" || pythonAtLeast "3.13";
```

```suggestion
  disabled = pythonOlder "3.8";
```
Upstream supports 3.13 https://gitlab.com/engmark/vcard/-/commit/665f3e083aa26beb0d6080555e5d4c31b65f79c1

---

Thanks!